Bottom: dbdd058e44503c17c9505cf7c3cd6e57c950b41e
Top:    ad24fd2b472eec95ebc510c5703c3693e1f95396
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-23 10:51:51 +0800

vhost: bypass wrap counter in SET/GET_VRING_BASE

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 3513e92..e5bc4bb 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -1415,10 +1415,12 @@ long vhost_vring_ioctl(struct vhost_dev *d, unsigned int ioctl, void __user *arg
 			break;
 		}
 		vq->last_avail_idx = s.num;
+#if 0
 		if (vhost_has_feature(vq, VIRTIO_F_RING_PACKED)) {
 			vq->last_avail_wrap_counter = s.num >> 31;
 			vq->avail_wrap_counter = vq->last_avail_wrap_counter;
 		}
+#endif
 		/* Forget the cached index value. */
 		vq->avail_idx = vq->last_avail_idx;
 		break;
